### DISCUSSION TITLE
Реалистичные аснп (и модели, и прошивки, разные виды и прочее)

### DISCUSSION THREAD

**Discussion by CralixRaev** [metrostroi-repo/MetrostroiAddon#206] [Ideas]
Почему бы не сделать аснп более реалистичными? В данный момент АСНП вообще не похожа на реальность (как минимум, я таких не видел, я не эксперт))), поэтому можно было бы внести реалистичности в данную систему, а так же сделать несколько видов с разными моделями и прошивками. Модели готов запилить/предоставить (учитывая, что я недавно делал, правда на скринах беды s bashkoyu а точнее текстурами)
![image](https://user-images.githubusercontent.com/32234868/104166725-48a44d00-540c-11eb-9fd5-c19fc419896b.png)
![image](https://user-images.githubusercontent.com/32234868/104166794-5e197700-540c-11eb-8b09-8dff7c553529.png)
я думаю товарищ ImPerzz сможет правильней рассказать о АСНП

**Comment by CrIcKeT98** [comment]
Перз, разлогинься XD

**Comment by ImPerzz** [comment]
Так пацаны, че за движ, я в деле

**Comment by CralixRaev** [comment]
> Так пацаны, че за движ, я в деле

с тебя инфа и прошивки, с нас модели аххаха)

**Comment by glebqip** [comment]
Текущий информатор - можно сказать самодельный. За модель была как я понял взят какой то единичный вариант АСНП, а по скриптам просто рандом/переделка очень очень первой версии информатора. Сделать как реальную можно, но какой-то один-два варианта. Выбор из 5 разных вариаций АСНП не очень наверное прикольно

**Comment by alexandr-sizov** [comment]
Если делать, то я согласен с Глебом, 2-3 версии, к примеру для Ежа/Еобразных, ПНМ в 4 блоке и ПНМ у вольтметров и не городить к этому еще 10050 прошивок
